creator-node/src/logging.js

```diff
@@ -97,20 +97,12 @@ function loggingMiddleware(req, res, next) {
 }
 
 /**
- * Add fields to a child logger instance
- * @param {*} req
- * @param {Object} options fields to add to child logger
- * @returns a logger instance
+ * Creates and returns a bunyan childLogger
+ *
+ * Options object adds properties to the JSON logger, allowing for better log filtering/querying
  */
-function setFieldsInChildLogger(req, options = {}) {
-  const fields = Object.keys(options)
-
-  const childOptions = {}
-  fields.forEach((field) => {
-    childOptions[field] = options[field]
-  })
-
-  return req.logger.child(childOptions)
+logger.createChildLogger = function (options = {}) {
```

Comment:
since this is on the logger object we can create a child logger of a child logger right? i think that's  why the req property was there but pretty sure we never did that